scripts/test

```diff
@@ -1,3 +1,7 @@
 #! /bin/bash
 set -xeuo pipefail
+
+# Necessary to install timoni for testscript execution
+make go-deps
```

Comment:
Note that we're running `make go-deps` now during the test script. Total execution time of installing deps and running the tests is about 3 minutes.